### PR TITLE
Add possessive functionality, handling plural/singular cases.

### PR DESCRIPTION
**Pull request by AwesomeBobX64**
Had a conversation with a co-worker about the utility of this library and how we'd need only add a bit of our own functionality to achieve proper handling of possessive punctuation for plural and singular words. Then I figured, why not just contribute straight to the library?

Here's the result.

### CODE REVIEW

**Comment by AwesomeBobX64** [comment]
Singular words that end in “s” require a ‘s versus just a ‘.

I’ll link Grammarly once I get back to my laptop :)

**Comment by AwesomeBobX64** [comment]
So swap the predicates in the ternary? Easy fix. I’ll give it a shot in the AM.

Happy to contribute! :)

**Comment by AwesomeBobX64** [comment]
Just checking in on this :) Ready to merge?

**Comment by AwesomeBobX64** [comment]
Can this be merged?

**Comment by blakeembrey** [comment]
@AwesomeBobX64 I think this is better as a separate library, not in `pluralize`. If you want to create it, I'll link to it, but this isn't something I think I can or want to maintain.

File: pluralize.js [hunk 1]
```diff
@@ -179,6 +179,21 @@
...
 
+  /**
+   * Make a word posessive handling plurals properly.
+   *
+   * @type {Function}
+   */
+  pluralize.possessive = function (word) {
+    var possessiveWord = word + "'s";
+
+    if (word.endsWith('s') && pluralize.isPlural(word)) {
```
Comment (blakeembrey) [inline pluralize.js]: Also note that `endsWith` limits the JavaScript engines this can run on and changes compatibly in a breaking way of this package.

Comment (AwesomeBobX64) [reply pluralize.js]: I had no idea. Thanks :)

**Pull request closed without merging**